Cargo.toml

```diff
@@ -26,9 +26,9 @@ serde_cbor = "0.11.2"
 tokio = { version = "1.28.1" }
 
 # zk-evm dependencies
-plonky2 = "0.2.0"
-mpt_trie = "0.1.1"
-evm_arithmetization = "0.1.1"
+plonky2 = { git = "https://github.com/0xPolygonZero/plonky2", branch = "main" }
+mpt_trie = { git = "https://github.com/0xPolygonZero/zk_evm", branch = "prep-release" }
+evm_arithmetization = { git = "https://github.com/0xPolygonZero/zk_evm", branch = "prep-release" }
```

Comment:
We should be doing the release soon, I'll update the deps once more here to point to newest version before merge.